packages/recode/build.sh

```diff
@@ -0,0 +1,8 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/pinard/Recode"
+TERMUX_PKG_DESCRIPTION="Charset converter tool and library"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="Marlin Sööse <marlin.soose@laro.se>"
+TERMUX_PKG_VERSION="3.7.8"
+TERMUX_PKG_SRCURL=https://github.com/rrthomas/recode/releases/download/v3.7.8/recode-${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
Please use $TERMUX_PKG_VERSION in the other place in the SRCURL as well

Reply:
Oops, thanks for catching this!